embedding-sdk/unauthenticated/readme.md

```diff
@@ -0,0 +1,84 @@
+# MongoDB Charts Embedding Example - Unauthenticated Embedded Chart
+
+## Background
+
+---
+
+_[See the MongoDB Charts Embedding Docs for more details](https://docs.mongodb.com/charts/saas/embedding-charts/)_
+
+MongoDB Charts allows you to create visualizations of your MongoDB data using a simple web interface. You can view the visualizations within the Charts UI, or you can use the Embedding feature to render the charts in an external web application. When you embed a chart, you can choose whether you want it to be Unauthenticated (meaning anyone who has the embed code can view the chart), or to use "Verified Signature" authentication (which checks the integrity of a signature in the URL before the chart will render).
+
+For the "Unauthenticated" option, you can copy the provided snippet and include it in any web site; no extra steps are required. This demo will provide an example on how to do exactly that, along with showing off the various ways your application can interact with the Embedded Charts API.
```

Comment:
I know this was copied from the existing samples, but we need to rewrite this. We shouldn't mention Verified Signature here since it will be deprecated. I'd suggest something like:

> MongoDB Charts allows you to create visualizations of your MongoDB data using a simple web interface. You can view the visualizations within the Charts UI, or you can use the Embedding feature to render the charts in an external web application. 

> Charts can be embedded either using a simple IFRAME snippet, or by using the Charts Embedding SDK from your JavaScript code. When using the SDK, embedded charts can be either unauthenticated (meaning anyone who has the embed code can view the chart), or authenticated (whereby the user can only view the chart if they have an active authentication session linked to a Charts authentication provider). 

> This sample shows how to use the JavaScript Embedding SDK to render unauthenticated embedded charts, along with showing off the various ways your application can interact with charts using the SDK.